docker_xylem/service.py

```diff
@@ -36,6 +37,7 @@ def __init__(self, config):
         self.current = {}
 
     def xylem_request(self, queue, call, data):
+        self.log.info('HTTP request to xylem.')
```

Comment:
Maybe we should add more detail here? The volume name seems like a good thing to know.